src/reducers/dependencies.reducer.test.js

```diff
@@ -388,6 +392,27 @@ Object {
 }
 `);
   });
+
+  test('reset to initialState on RESET_ALL_STATE action', () => {
+    const initialState = {
```

Comment:
I think it's better to rename the `initialState` import so we're not getting a collision if in a test it is called `initialState`. But that's a personal opinion.

You're right `prevState` is better then `initialState`. First I had `someState` but I think `prevState` is better.
I would keep the renaming to `dependenciesInitialState` but using `initialState` would be also OK. Then we have to avoid `initialState` and only use it for the very first state e.g. `projects.reducer` is also using `initialState` in line 214 (just to mention an occurence).
I think it's better to tackle this after this PR if you'd like to remove the renaming of `initialState`. If we keep the renaming I can add the thing with `prevState` for the first state in my added test case.